contrib/test_jenkins.sh

```diff
@@ -1132,7 +1132,7 @@ run_test_proto_disable() {
 run_asan_check() {
 	build devel --enable-gtest --enable-asan --without-valgrind
 
-	if ! ldd ${ucx_inst}/src/tools/info/.libs/ucx_info | grep -q "libasan.so"
+	if ! ldd ${WORKSPACE}/test/gtest/gtest | grep -q "libasan.so"
```

Comment:
i think need to check for gtest in ${ucx_inst}

Reply:
done